### PR TITLE
Removed client & scripts code

### PR DESCRIPTION
**Pull request by KTong821**
## Purpose
This removes the entirety of the clients folder, as well as the population scripts used to create new clubs and events. These were carried over during the Waterpark codebase clone, and should be removed now that they are no longer necessary.

### CODE REVIEW

**Review by xujustinj — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,14 +1,78 @@
-# Auth
+# Waterpark Server
```
Comment (xujustinj) [inline README.md:1]: TODO: update this to relate to `auth`. Not sure if that's something we want to add to this PR.

File: package.json [hunk 2]
```diff
@@ -1,15 +1,64 @@
 {
-  "name": "@loolabs/waterpark",
-  "description": "Loo Labs' centralized clubs and events platform",
+  "name": "@loolabs/waterpark-server",
+  "description": "REST API server for Loo Labs' centralized clubs and events platform",
```
Comment (xujustinj) [inline package.json]:
```suggestion
  "name": "@loolabs/auth",
  "description": "The Loo Labs central SSO/OIDC auth micro-service, usable by all Loo Labs apps",
```

**Review by xujustinj — APPROVED** [review]
small changes

**Review by ShreyasPrasad — APPROVED** [review]
lgtm

**Pull request closed without merging**